flavors/linuxlocal/local_flavor.go

```diff
@@ -30,13 +31,15 @@ import (
 
 // Flavor glues together multiple plugins to mange VPP and linux interfaces configuration using local client.
 type Flavor struct {
+	Local  local.FlavorLocal
```

Comment:
1. use anonymous field name
2. please rename also Flavor to FlavorVppLocal (to avoid anonymous composition conflicts)